### PR TITLE
Added "strategies" dropdown in nav

### PR DESCRIPTION
**Pull request by Jordan-George**

### CODE REVIEW

File: js/exhibit.js [hunk 1]
```diff
@@ -152,5 +152,11 @@ $(document).ready(function() {
       $('.repos-count').html('Showing ' + index + ' of ' + total)
     });
   }
+
+  $(".dropdown-button").dropdown(
+    { hover: true }
+  );
+
+  $(".button-collapse").sideNav(function() {});
```
Comment (vabarbosa) [inline js/exhibit.js]: you may call  `$(".button-collapse").sideNav()`  if you do not want to pass any options

**Review by vabarbosa — APPROVED** [review]
nice work @Jordan-George

**Comment by vabarbosa** [comment]
@mikebroberg @Jordan-George are we doing away with the fixed navbar? reason i ask, is it is no longer fixed to the top in this branch just wanted to make sure it was intentional.

**Review by vabarbosa — COMMENTED** [review]
@Jordan-George @mikebroberg we probably want to verify the title for the features (i.e., strategies). for example `5-ourwork.md` does not have a title and thus appears in the Strategies drop down as **5 Ourwork**

**Comment by vabarbosa** [comment]
@Jordan-George @mikebroberg ... i have made the additional changes mentioned in my above comments and merged changes into master (also merged pr #8). please sync/update with master to get all the latest & greatest

**Comment by Jordan-George** [comment]
The nav bar not working wasn't intentional. Thanks for fixing it! @mikebroberg said that those titles after the first four were temporary.

**Pull request merged**